platform/android/src/thread.cpp

```diff
@@ -33,5 +36,19 @@ void makeThreadLowPriority() {
     setpriority(PRIO_PROCESS, 0, 19);
 }
 
+ThreadData attachThread() {
```

Comment:
I don't think we need the `ThreadData` because it could be stored on a `thread_local` on Android.

Reply:
To be clear, I'm talking about we don't need `ThreadData` on the function signature.